### PR TITLE
Fix formatting

### PR DESCRIPTION
**Pull request by JoshAtticus**
Minor formatting correction

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ Before creating your own bot using Octavia's source code, you will need to creat
 
 To run your bot, simply use
 > python3 main.py
```
Comment (MikeDev101) [inline README.md:22]: I should've used code tags (`python3 main.py`) instead.

Comment (JoshAtticus) [reply README.md:22]: probably, I can do that

**Comment by JoshAtticus** [comment]
Changed to code tags

File: README.md (hunk 1 above)
Comment (JoshAtticus) [reply README.md]: > I should've used code tags (`python3 main.py`) instead.

Done

**Review by MikeDev101 — APPROVED** [review]

**Pull request merged**